### DISCUSSION TITLE
Post-release v12.2.0 task list

### DISCUSSION THREAD

**Discussion by aleclarson** [radashi-org/radashi#284] [Ideas]
Now that v12.2.0 has been released, I'm using this discussion to track some things I'd like to do next.

If anyone wants to lend a hand, let us know here, and remember that I'm here to answer any questions you might have.

- [x] Publish v12.2.0 to [jsr.io](https://jsr.io/@radashi-org/radashi) (and get the automated publish script working)
- [ ] Add banner to website announcing the v12.2.0 release. Point to [this link](https://github.com/orgs/radashi-org/discussions/285).
- [x] Fix the “nightly release” GitHub action (authorization issue with `git push`). 🔥 *High priority*
- [ ] Set up a GitHub action for auto-deploying the [radashi-docs](https://github.com/radashi-org/radashi-docs) repository when a change is made (either directly or to the `docs/` folder in the main [radashi](https://github.com/radashi-org/radashi/tree/main/docs) repository). 🔥 *High priority*
- [x] Add `prerelease` label to finished PRs for v12.3.0 beta. 🔥 *High priority*
- [ ] Remove the CJS build now that Node.js has [added](https://github.com/nodejs/node/pull/51977) `--experimental-require-module`. 🐌 *Low priority*
- [ ] Set up a section for release notes on the website. 🐌 *Low priority*
  - Write an `.astro` file that fetches the files from the [release-notes branch](https://github.com/radashi-org/radashi/tree/release-notes).

**Comment by MarlonPassos-git** [comment]
I could help with some things

Regarding "Add banner to website announcing the v12.2.0 release. Point to [this link](https://github.com/orgs/radashi-org/discussions/285)," do you have any examples of this type of banner?

Another thing: what do you think about creating release posts similar to what you did on this page: https://radashi.js.org/blog/vs-radash/ or like how Jest does it: https://jestjs.io/pt-BR/blog/2022/08/25/jest-29? We don’t need to create something specifically for a blog—just publish the markdown on that page.
